apistructs/job.go

```diff
@@ -111,6 +111,7 @@ type JobFromUser struct {
 	BackoffLimit            int                    `json:"backoffLimit,omitempty"`
 	Params                  map[string]interface{} `json:"params,omitempty"`
 	TaskContainers          []TaskContainer        `json:"taskContainers"`
+	Network                 map[string]string      `json:"network,omitempty"`
```

Comment:
Use typed struct is better. Or you need to clarify all the selectable values.

Reply:
refactor network to type, and impl `IsHostMode` for k8s executor